haystack/json-schemas/generate_schema.py

```diff
@@ -0,0 +1,21 @@
+import os
+import logging
+import sysconfig
+from pathlib import Path
+
+from haystack.nodes._json_schema import update_json_schema
+
+logger = logging.getLogger(__file__)
+
+try:
+    logger.warning(f"Generating the YAML schema for Haystack Pipelines... ")
```

Comment:
not sure this should be a warning, I don't disagree in principle but the output I got is a bit ugly:
```
WARNING:/opt/homebrew/Cellar/python@3.10/3.10.6_1/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site.py:Generating the YAML schema for Haystack Pipelines...
```

Reply:
Woah it didn't look like this on my machine :smile: Fixing

Reply:
I made it a warning because it can take a perceptible amount of time, so I think users should be warned about why.

Reply:
Ok, since the message is only visible _after_ Python starts, what if we change it to something like this:
```
logger.warning(f"Haystack generated the YAML schema to support loading Pipelines, this only happens once, after installing the package. ")
```